### ISSUE TITLE
Jump to recent messages lacks accessibility semantics

### ISSUE THREAD

**Issue by daniel-montalvo** [thelounge/thelounge#4862]
Once you select a channel on the network and new messages appear, there is a button to go to the most recent messages.

```
<div class="scroll-down tooltipped tooltipped-w tooltipped-no-touch" aria-label="Jump to recent messages"><div class="scroll-down-arrow"></div></div>
```

Adding appropriate semantics and keyboard support is recommended. You can set this to be a button (ideally using native HTML `button`) so that it inherits all the browser's built-in keyboard support.
